components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.ts

```diff
@@ -152,6 +151,33 @@ export class PolicyAddMembersComponent implements OnInit, OnDestroy {
           // Now that the membersAvailableMap is correct, refresh.
           this.refreshSortedMembersAvailable();
       });
+
+    this.store.select(addPolicyMembersStatus).pipe(
+      takeUntil(this.isDestroyed),
+      filter(addState => this.addingMembers && !pending(addState)))
+      .subscribe((addPolicyState) => {
+        if (addPolicyState === EntityStatus.loadingSuccess) {
+          this.addingMembers = false;
+          this.router.navigate(this.backRoute(), { fragment: 'members' });
+        }
+      });
+
+    combineLatest([
+      this.store.select(addPolicyMembersStatus),
+      this.store.select(addPolicyMembersHTTPError)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      filter(() => this.addingMembers),
+      filter(([addState, error]) => addState === EntityStatus.loadingFailure && !isNil(error)))
+      .subscribe(([_, error]) => {
+        this.addingMembers = false;
+        if (error.message === undefined) {
+          this.addMembersFailed = 'An error occurred while attempting ' +
+            'to add members. Please try again.';
+        } else {
+          this.addMembersFailed = `Failed to add members: ${error.message}`;
```

Comment:
`error.message` returns the top level `message` but that's not the API error message. for that we want the `message` nested under the top-level `error` object. so i would change the `error` variable to `resp` or something then have the string use `resp.error.message`
<img width="428" alt="Screen Shot 2019-11-15 at 11 49 04 AM" src="https://user-images.githubusercontent.com/21015366/68971340-224e3780-079e-11ea-951d-22249777c7de.png">

Reply:
Fixed!